perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -1080,6 +1080,248 @@ public void getVoRichAdminsWithWrongObjects() throws Exception {
 		AuthzResolver.getRichAdmins(sess, testUser, new ArrayList<>(), Role.VOADMIN, false, true);
 	}
 
+	@Test
+	public void getVosWhereUserIsInRoles() throws Exception {
+		System.out.println(CLASS_NAME + "getVosWhereUserIsInRoles");
+
+		final Vo testVo = perun.getVosManager().createVo(sess, new Vo(0,"testvo1","testvo1"));
+		final Vo testVo2 = perun.getVosManager().createVo(sess, new Vo(1,"testvo2","testvo2"));
+		final Vo testVo3 = perun.getVosManager().createVo(sess, new Vo(2,"testvo3","testvo3"));
+		final Group testGroup = perun.getGroupsManager().createGroup(sess, testVo, new Group("testGroup", "testg"));
+		final Member testMember = createSomeMember(testVo);
+		final User testUser = perun.getUsersManagerBl().getUserByMember(sess, testMember);
+		perun.getGroupsManager().addMember(sess, testGroup, testMember);
+
+		AuthzResolver.setRole(sess, testUser, testVo, Role.VOADMIN);
+		AuthzResolver.setRole(sess, testUser, testVo2, Role.SPONSOR);
+		AuthzResolver.setRole(sess, testGroup, testVo3, Role.VOOBSERVER);
+		List<Vo> result = AuthzResolver.getVosWhereUserIsInRoles(sess, testUser, Arrays.asList(Role.VOADMIN, Role.VOOBSERVER));
+
+		assertEquals(2, result.size());
+		assertTrue(result.containsAll(Arrays.asList(testVo, testVo3)));
```

Comment:
Nit: For assertions like this, it is good to use the Assertj library. You don't even need to check the size explicitly with such assertion.
```suggestion
		assertThat(result).containsOnly(testVo, testVo3);
```

Reply:
Thanks. I will use it next time but I will not change it in this PR.